charts/cluster-api-runtime-extensions-nutanix/templates/virtual-ip/kube-vip/manifests/kube-vip-configmap.yaml

```diff
@@ -46,7 +46,7 @@ data:
             - name: vip_retryperiod
               value: "2"
             - name: address
-              value: '{{ `{{ .ControlPlaneEndpoint.Host }}` }}'
+              value: '{{ `{{ .Address }}` }}'
```

Comment:
curious if and how this address is added to the SANs in the certificates referred by the kubeconfig

Reply:
Great question, at first I thought that would be necessary too but it wasn't.
It's not needed because this IP is not used directly by any clients when reaching the API server, everything goes through the controlPlaneEndpoint.